assembly/src/parsers/ast/field_ops.rs

```diff
@@ -0,0 +1,65 @@
+use crate::validate_operation;
+
```

Comment:
nit: let's remove a blank line after this import.

Reply:
fixed